lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -45,7 +49,9 @@
                                 model: userAccountModel,
                                 title: gettext('Email Address'),
                                 valueAttribute: 'email',
-                                helpMessage: gettext('The email address you use to sign in to edX. Communications from edX and your courses are sent to this address.')
+                                helpMessage: interpolate_text(
+                                    gettext('The email address you use to sign in to {platform_name}. Communications from {platform_name} and your courses are sent to this address.'), {platform_name: platformName}
```

Comment:
We could get rid of "to {platform_name}" in the first sentence, but we should leave it in the second sentence.